### ISSUE TITLE
general - Unable to use 0x libraries: 'cannot read property slice of undefined'

### ISSUE THREAD

**Issue by the-jackalope** [0xProject/0x-monorepo#2041]
Doing almost anything that makes requests to the network with the latest release of various 0x libraries results in the same error. The easiest to demonstrate is the `contract-wrappers` repo - I expect to retrieve a public address's ZRX token balance on the mainnet with the following code:

```
const Web3 = require('web3');
const provider = new Web3.providers.HttpProvider('http://provider-url-here');

const { ContractWrappers } = require('@0x/contract-wrappers');
const  contractWrappers = new ContractWrappers(provider, { networkId: 1 });

contractWrappers.erc20Token.getBalanceAsync('ZRX_ADDRESS_HERE', 'ANY_ETHEREUM_ADDRESS_HERE').then(console.log).catch(console.error)
```

Which consistently yield the following error:
```
> TypeError: Cannot read property 'slice' of undefined
    at Function.BaseContract._throwIfRevertWithReasonCallResult (/Users/johngcook/Developer/fetch/desktop/node_modules/@0x/base-contract/lib/src/index.js:169:27)
    at ERC20TokenContract.<anonymous> (/Users/johngcook/Developer/fetch/desktop/node_modules/@0x/contract-wrappers/node_modules/@0x/abi-gen-wrappers/lib/src/generated-wrappers/erc20_token.js:428:62)
    at step (/Users/johngcook/Developer/fetch/desktop/node_modules/@0x/contract-wrappers/node_modules/@0x/abi-gen-wrappers/lib/src/generated-wrappers/erc20_token.js:56:23)
    at Object.next (/Users/johngcook/Developer/fetch/desktop/node_modules/@0x/contract-wrappers/node_modules/@0x/abi-gen-wrappers/lib/src/generated-wrappers/erc20_token.js:37:53)
    at fulfilled (/Users/johngcook/Developer/fetch/desktop/node_modules/@0x/contract-wrappers/node_modules/@0x/abi-gen-wrappers/lib/src/generated-wrappers/erc20_token.js:28:58)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
```

Whereas I expect to get the address's token balance. I don't believe it to be an issue with the provider as I can instantiate a Web3 instance with the provider and use it to correctly query the address's ETH balance.

@0x/contract-wrappers: `10.1.0`
web3: `1.0.0-beta.55`

**Comment by DonGambas** [comment]
I'm experiencing the same exact issue.

**Comment by fabioberger** [comment]
Thanks for the report @the-jackalope @DonGambas we are going to work on addressing this ASAP.

**Comment by DonGambas** [comment]
@fabioberger thanks for the response, I initially installed the packaged with npm, when I installed them with yarn there was no issue 🤦‍♂ 

I have one additional question re: the upgrade guide. All of the order validation functions got moved to the order-utils package. It would be helpful to see an example of how to use the order validation utils in the order-utils package in particular how to set up the `exchangeTradeEmulator`.
